composer/trainer/trainer_hparams.py

```diff
@@ -185,6 +185,8 @@ class TrainerHparams(hp.Hparams):
             .. seealso:: :mod:`composer.callbacks` for the different callbacks built into Composer.
         load_path (str, optional): See :class:`.Trainer`.
         load_object_store (ObjectStore, optional): See :class:`.Trainer`.
+        load_logger_destination (LoggerDestination, optional): Used to specify a ``LoggerDestination`` for
```

Comment:
Can you mention that it should not be specified with` load_object_store`, and similar thing on the `load_object_store` docstring line?